admiral.Rproj

```diff
@@ -1,4 +1,5 @@
 Version: 1.0
```

Comment:
Hey @jimrothstein can you revert this change for this file please

Reply:
should be fixed.